dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -716,7 +716,15 @@ void CommandBarFlyoutCommandBar::EnsureAutomationSetCountAndPosition()
     {
         if (auto commandAsUIElement = command.try_as<winrt::UIElement>())
         {
-            if (commandAsUIElement.Visibility() == winrt::Visibility::Visible)
+            // Don't count AppBarSeparator if IsTabStop is false
+            if (auto separator = commandAsUIElement.try_as<winrt::AppBarSeparator>())
+            {
+                if (!separator.IsTabStop())
+                {
+                    continue;
+                }
+            }
+            else if (commandAsUIElement.Visibility() == winrt::Visibility::Visible)
             {
                 sizeOfSet++;
             }
```

Comment:
I don't see why you'd want to special case AppBarSeparator here - wouldn't anything with IsTabStop=false be excluded?  And there's more things that affect focusability than just IsTabStop, like IsEnabled and AllowFocusWhenDisabled - should those be respected too?

Reply:
@llongley to explain why IsTabStop is changed to true in `CommandBarFlyoutCommandBar::FocusControl`

Reply:
It looks like this stems from this design:

https://github.com/microsoft/microsoft-ui-xaml/issues/339

History tells me that @RBrid was the one who implemented it.  It looks like that might be where the issue lies - does it fix the issue to just check for AppBarSeparator in `CommandBarFlyoutCommandBar::OnKeyDown` and pass over instances of it when considering where to set focus to?

Reply:
I will not make change on OnKeyDown. `Tab` is handled by the framework after secondary menus are open